modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -5153,6 +5153,7 @@ do-not-repeat=Do Not Repeat
 do-not-repeat-this-event=Do not repeat this event.
 do-not-show-me-this-again=Do not show me this again.
 do-not-show-me-this-again-help=If you connect your Liferay DXP instance to Liferay Analytics Cloud in the future, the panel will show up again.
+do-not-show-this-again=Do not show This again
```

Comment:
When running `gw buildLang`, `this` -> `This`. I don't know what is going on 🤷🏼

Reply:
Try to end the sentence with a dot

Reply:
ohhh, ok!

Reply:
Hey, I noticed `do-not-show-me-this-again -> Do not show me this again.`, I'll use this for more consistency around DXP

Reply:
https://github.com/liferay-frontend/liferay-portal/pull/2164/commits/52b6c21e0e2ec6fcaee9361b085f825af0f9327d